pyproject.toml

```diff
@@ -1,10 +1,13 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
[consider] All the `.github` CI `.yaml` files have no license identifier.

Reply:
That's the same for OpenAssetIO proper, i've followed that example.